src/isIsogram.test.js

```diff
@@ -3,7 +3,27 @@
 describe('isIsogram', () => {
   const { isIsogram } = require('./isIsogram');
 
-  it(`should be declared`, () => {
-    expect(isIsogram).toBeInstanceOf(Function);
+  it(`should be declared`, () => expect(isIsogram).toBeDefined());
+
+  it(`returns 'true' for word without repeating letters`, () => {
+    expect(isIsogram('playgrounds')).toBe(true);
+  });
+
+  it(`returns 'false' for word with repeating letters`, () => {
+    expect(isIsogram('look')).toBe(false);
   });
+
+  it(
+    `returns 'false' for word with repeating upper and lower case letters`,
+    () => expect(isIsogram('Adam')).toBe(false),
+  );
+
+  it(`returns 'true' for empty string`, () => {
+    expect(isIsogram('')).toBe(true);
+  });
+
+  it(
+    `returns 'false' for word with repeating letters standing together`,
+    () => expect(isIsogram('Oops')).toBe(false),
```

Comment:
Similar to the previous issue, there is a syntax error here. The arrow function should not end with a comma. Replace the comma with a closing parenthesis to fix the syntax.